debian/changelog

```diff
@@ -1,3 +1,112 @@
+
+    [Changes recorded in 2.4.0-rc2 tag]
+    - Only report ready after first apply() completes.
+    - Add a grace period before deleting routes.
+    - Leave interfaces in the route cleanup grace period dirty, and add tests
+
+    [Changes recorded in 2.4.0-rc1 tag]
+    - Skip recalculation of selector matches if selector hasn't changed (#1482).
+    - Use updated Typha client API (#1484).
+    - Improve testing and test coverage (#1486, #1494, #1496, #1497).
+    - Make test suites produce junit reports (#1488).
+    - Allow selection of policy allow action (#1492).
+    - Implement liveness and readiness endpoints for Felix (#1489).
+    - Improve Calico version reporting (#1499).
+    - Streamline conntrack state deletions (#1500, #1498).
+    - Add release note to PR template (#1502).
+    - Add support for multiple CIDRs in a match rule (#1483, #1505).
+    - Support using a lock to coordinate iptables programming with other
+      software (#1491, #1504).
+    - Move logutils functionality to libcalico-go (#1503).
+    - Add pre-DNAT policy support (#1506).
+    - Update glide pin for logrus (#1509).
+    - Allow for time fuzziness in route table UT (#1510).
+    - Update to Typha v0.3.0 (#1512).
+
+ -- Neil Jerram <neil@tigera.io>  Fri, 04 Aug 2017 22:06:28 +0000
+
+felix (2.3.0~__STREAM__) __STREAM__; urgency=low
+
+  * Felix 2.3.0 (from Git commit 85f9fff).
+    [Changes recorded in 2.3.0 tag]
+    This is a feature release of Felix, Calico's per-host agent.
+
+    The headline feature in this release is a significant increase in scale when using the
+    Kubernetes datastore driver by introducing support for a new daemon, Typha. Typha
+    connects to the Kubernetes API server and fans out updates to a number of Felixes.
+    By having only a handful of Typha instances connected to the API server instead of
+    many Felixes, we place a lot less load on the API server.  In addition, Typha is able
+    to squash unimportant updates form the API server, significantly reducing the
+    number of mesages each Felix has to handle.
+
+    Typha is disabled by default. The documentation for how to configure Typha and Felix
+    together will follow as part of the wider Calico 2.3.0 meta-release.
+
+    This release also contains a number of minor enhancements:
+
+    - Performance: Scan iptables-save output incrementally when calculating hashes.
+    - Performance: Disable WithFields logging in the ipsets resync parse loop.
+    - Performance: Disable some WithField calls in the inner iptables resync loop.
+    - Allow the iptables post-write check interval to be set.
+    - Improve diagnostics around ipset restore failures
+    - Log ip6tables-save stderr if it fails
+    - Convert interval config parameters to time.Duration for increased precision.
+    - Rev libcalico-go to v1.4.4 and Typha to v0.2.2.
+    - Make JUST_A_MINUTE the default, as it's more useful for devs
```

Comment:
2.3 again, as you say.  I think it would be confusing if this was any different from the 2.3 notes that we've already merged into 2.3.x-series and master.